### PR TITLE
Rm id use

### PR DESCRIPTION
**Pull request by marimeireles**
Hello,
This is an approach on removing `id`s. Comments are appreciated!
(I was building on top of Antonio's fix, will rebase later =)

### CODE REVIEW

**Comment by tedpatrick** [comment]
There are several ways to find elements in the DOM. 

```js
// find the item (singular) where id="foo"
document.getElementById("foo") 

// find items (plural) where pys-id="foo"
document.querySelectorAll('[pys-id="foo"]')  
```

If we have our own distinct attribute for pyscript ids, we will never collide with `id` at all.

Benefits:
1. Tests can utilize this
2. Internal code can utilize this
3. No side-effects outside of PyScript

**Comment by marimeireles** [comment]
It's a good idea. Thanks @tedpatrick! So that's the current plan? 

It still doesn't eliminate the problems that might be caused by dup. `id` etc..

I still have some other questions:
How about the order these are generated? Is what I pointed okay? As I think more on this case I think it's fine and it makes sense to keep the number order equal to when the elements were generated rather than corresponding to their depth in the DOM tree.

**Comment by tedpatrick** [comment]
If we standardize on using attributes other than id, we do not pollute
the id namespace at all. If there is a single collision,
getElementById breaks and it makes things unpredictable.

I am not sure how best to implement this, I was simply proposing an
alternative to using the "id" attribute.

It also feels like the original <py-repl> element should have child
elements not generate sibling elements. If elements are nested, they
can have scroll behavior applied or style applied as needed.

**Comment by marimeireles** [comment]
Hum, I don't understand what you mean.
They're not generating sibling elements as far as I understand the elements generated are children:

Screenshot 2022-10-31 at 18 32 42

**Comment by marimeireles** [comment]
Testing this code over this snippet of example:
```
<py-script id="first">
    display('hello 1')
</py-script>
<py-script id="second">
    display('hello 2')
</py-script>
<py-script id="third">
    display('hello 3')
</py-script>
```
I get the same errors I got last week, I thought Antonio's latest PR had fixed that but not really...
Essentially there's something wrong with the way `get_current_display_target`.
If you run the above code on this PR you'll notice that the `self._element` variable of the `Element` object is not being updated.
In this case it's always the last `div` that's passed to the `write` method, causing `write` to overwrite the previous outputs.

**Comment by antocuni** [comment]
> It also feels like the original <py-repl> element should have child elements not generate sibling elements. If elements are nested, they can have scroll behavior applied or style applied as needed.

+1 for this. I think I proposed it somewhere but I can no longer find my original text.
Currently, a `<py-repl auto-generate=true>` generates more of less this code:

```html
<body>
    <py-repl auto-generate="true" exec-id="1"> 
        <div class="py-repl-editor"> ... </div>
        <div class="py-repl-output"> ... </div>
    </py-repl>
    <py-repl auto-generate="true" exec-id="2"> ... </py-repl>
    <py-repl auto-generate="true" exec-id="3"> ... </py-repl>
</body>
```

We should do this instead:
```html
<body>
    <py-repl auto-generate="true"> 
        <div class="py-repl-cell" py-cell-id="1">
            <div class="py-repl-editor"> ... </div>
            <div class="py-repl-output"> ... </div>
        </div>
        <div class="py-repl-cell" py-cell-id="2">
            ...
        </div>
        <div class="py-repl-cell" py-cell-id="3">
            ...
        </div>
    </py-repl>
</body>
```

Also, while we are at it, we should also consider to make `auto-generate=true` the default behavior, because it makes more sense.
And probably find another name for turning off the behavior; I'm happy to hear more proposals for a name, but a good starting point could be `loop="false"` (because the L in repL stands for "loop").
So:
```html
<py-repl></py-repl> <!-- this generates news cells automatically -->
<py-repl loop="false"></py-repl> <!-- this doesn't -->
```

File: pyscriptjs/tests/integration/test_zz_examples.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def test_panel_stream(self):
         assert self.page.title() == "PyScript/Panel Streaming Demo"
         wait_for_render(self.page, "*", "<div.*?class=['\"]bk-root['\"].*?>")
 
-    @pytest.mark.skip("flaky test, see issue #864")
+    # @pytest.mark.skip("flaky test, see issue #864")
```
Comment (FabioRosado) [inline pyscriptjs/tests/integration/test_zz_examples.py:259]: This test isn't flaky anymore? Should we remove this line? 😄

**Review by FabioRosado — APPROVED** [review]
Left a small comment but looks good 😄

File: pyscriptjs/tests/integration/test_zz_examples.py (hunk 1 above)
Comment (marimeireles) [reply pyscriptjs/tests/integration/test_zz_examples.py:259]: Yeah, I am supposed to test this today.
Obrigada ✨

**Comment by marimeireles** [comment]
Thanks for the reviews folks!

**Pull request merged**